cider/exec/nextgen/jitlib/base/JITValueOperations.h

```diff
@@ -0,0 +1,62 @@
+#ifndef JITLIB_BASE_JITVALUEOPERATIONS_H
+#define JITLIB_BASE_JITVALUEOPERATIONS_H
+
+#include <type_traits>
+#include "exec/nextgen/jitlib/base/JITValue.h"
+#include "exec/nextgen/jitlib/llvmjit/LLVMJITFunction.h"
+
+namespace jitlib {
+template <typename T>
+struct is_jitvalue_convertable {
+  using NT = typename std::remove_reference<T>::type;
+  static constexpr bool v = std::is_arithmetic_v<T> || std::is_same_v<T, bool>;
```

Comment:
`NT`?

Reply:
NativeType, will fix it.

Reply:
I mean in `static constexpr bool v = std::is_arithmetic_v<T> || std::is_same_v<T, bool>;`, you does not use the `NativeType`.